public/js/main.js

```diff
@@ -1,222 +1,193 @@
-        datasets: [
-            {
-                label: 'My First dataset',
-                fillColor: 'rgba(3, 88, 106, 0.2)',
-                strokeColor: 'rgba(3, 88, 106, 0.80)',
-                pointColor: 'rgba(3, 88, 106, 0.80)',
-                pointStrokeColor: '#fff',
-                pointHighlightFill: '#fff',
-                pointHighlightStroke: 'rgba(220,220,220,1)',
-                data: [65, 59, 90, 81, 56, 55, 40]
-            },
-            {
-                label: 'My Second dataset',
-                fillColor: 'rgba(38, 185, 154, 0.2)',
-                strokeColor: 'rgba(38, 185, 154, 0.85)',
-                pointColor: 'rgba(38, 185, 154, 0.85)',
-                pointStrokeColor: '#fff',
-                pointHighlightFill: '#fff',
-                pointHighlightStroke: 'rgba(151,187,205,1)',
-                data: [28, 48, 40, 19, 96, 27, 100]
-            }
-        ]
-    };
-
+    // // Population by race
     $(document).ready(function () {
-        window.myRadar = new Chart(document.getElementById('canvas_radar').getContext('2d')).Radar(radarChartData, {
-            responsive: true,
-            tooltipFillColor: 'rgba(51, 51, 51, 0.55)'
-        });
+        var populationByRace = {
+            labels: ['White', 'Black', 'Asian', 'Multiple Races', 'Other', 'American Indian', 'Pacific Islander'],
+            datasets: [
+                {
+                    backgroundColor: '#316BBF',
+                    data: [982511, 532025, 57948, 57205, 27772, 6790, 2059]
+                }
+            ]
+        };
+
+        new Chart($('#populationByRace'), {
+            type: 'horizontalBar',
+            data: populationByRace
+        })
     });
 
-    var polarData = [
-        {
-            value: 300,
-            color: '#F7464A',
-            highlight: '#FF5A5E',
-            label: 'Red'
-        },
-        {
-            value: 50,
-            color: '#46BFBD',
-            highlight: '#5AD3D1',
-            label: 'Green'
-        },
-        {
-            value: 100,
-            color: '#FDB45C',
-            highlight: '#FFC870',
-            label: 'Yellow'
-        },
-        {
-            value: 40,
-            color: '#949FB1',
-            highlight: '#A8B3C5',
-            label: 'Grey'
-        },
-        {
-            value: 120,
-            color: '#4D5360',
-            highlight: '#616774',
-            label: 'Dark Grey'
-        }
-
-    ];
-
+    // School rating by enrollment
     $(document).ready(function () {
-        window.myPolarArea = new Chart(document.getElementById('canvas_area').getContext('2d')).PolarArea(sharePiePolorDoughnutData, {
-            responsive: true,
-            tooltipFillColor: 'rgba(51, 51, 51, 0.55)'
+        var ratingByEnrollment;
+
+        // Gather aggregates from Hercules
+        $.ajax({
+            method: 'GET',
+            url: 'http://hercules.code4hr.org/schools?groupBy=enrollment&averageField=gsRating',
```

Comment:
I chose gsRating because that one is 1-10 instead of 1-5... gave better graphics :)